packages/e2e-tests/specs/editor/various/block-hierarchy-navigation.test.js

```diff
@@ -46,7 +46,7 @@ describe( 'Navigating the block hierarchy', () => {
 		// Tweak the columns count.
 		await openDocumentSettingsSidebar();
 		await page.focus(
-			'.block-editor-block-inspector .components-range-control__number[aria-label="Columns"]'
```

Comment:
I was thinking this change makes the target selector less specific. I guess it's fine in this case because the block sidebar doesn't have any other input.type component, but I wanted to make sure I don't miss anything, as e2e tests tend to be fragile.